.github/workflows/ci.yml

```
@@ -45,11 +45,11 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        pyver: ["3.6", "3.7", "3.8", "3.9", "3.10"]
+        pyver: ["3.6.15", "3.7.15", "3.8.16", "3.9.16", "3.10.10"]
```

```suggestion
        pyver: ["3.7", "3.8", "3.9", "3.10", "3.11"]
```

I think this is the list of currently supported Python versions.